pages/index.tsx

```diff
@@ -14,38 +11,6 @@ type Props = {
 export default function Home({ session }: Props) {
   if (!session) return <MainSign />;
```

Comment:
Error: React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?  react-hooks/rules-of-hooks